src/python/rest/app.py

```diff
@@ -0,0 +1,32 @@
+# Simple flask app to serve the REST API
```

Comment:
same for flask - flask demo is great, lets have a separate template for it

Reply:
i like to have many example, for the more they are documented in the readme

Reply:
many examples is great to have in different repos - more over we have a dedicated repo for interoperability.
And we definitely need to have a very, very basic and simple repo with very basic examples (like here) to have the option to test fast with new versions and be sure that very basic functionality works fine.